blivet/populator/populator.py

```diff
@@ -453,7 +453,7 @@ def _populate(self):
 
         self.drop_lvm_cache()
 
-        if flags.installer_mode and not flags.image_install:
+        if flags.multipath_friendly_names and not flags.image_install:
```

Comment:
Yes, looking over it again, I can see how this might not be the best way to handle this. I don't really want to make a new flag just for that, so I suppose I can check flags.force_dev_updates